modules/oe_subscriptions_anonymous/src/Access/SubscriptionAccessCheck.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_subscriptions_anonymous\Access;
+
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Access\AccessResultInterface;
+use Drupal\Core\Routing\Access\AccessInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\flag\FlagInterface;
+use Drupal\flag\FlagServiceInterface;
+
+/**
+ * Checks access for a given subscription id.
+ */
+class SubscriptionAccessCheck implements AccessInterface {
+
+  /**
+   * Flag service.
+   *
+   * @var \Drupal\flag\FlagServiceInterface
+   */
+  protected FlagServiceInterface $flagService;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(
+    FlagServiceInterface $flagService) {
+    $this->flagService = $flagService;
+  }
```

Comment:
No need to break indent. If we are breaking indentation, then the format is
```
public function __construct(
  FlagServiceInterface $flagService
) {
  $this->flagService = $flagService;
}

Reply:
Fixed